### ISSUE TITLE
Was the ban a mistake?

### ISSUE THREAD

**Issue by MDBLeonardo** [ValveSoftware/Dota2-Gameplay#23395]
### Description

Hi Valve
I am writing to you to request a review of the penalty applied to my Dota 2 account, which was restricted due to game disconnections. I wish to clarify that these disconnections were not intentional, but rather a result of technical problems with my internet connection.

I understand the importance of maintaining a fair gaming experience and am sorry for any inconvenience cause d to other players. I am working to improve my connection and avoid future problems.I haven't done smurfing, I've been practicing with bots and that's proven in my history, I'm trying to improve so as not to cause problems for anyone in the game.

I hope you can address my problem and if not, anyway, thank you

thank you for your time

### Screenshots

![Captura de pantalla (3)](https://github.com/user-attachments/assets/eec6d866-e26b-425e-a6d5-096907bc4126)

**Comment by acebrzy** [comment]
hello, is your account okay now? we have the same issue :(

**Comment by MDBLeonardo** [comment]
No, mi cuenta sigue igual, pero estoy seguro de que es un error ya que
aveces me salen 2, uno permanente y otro asta 2038
